cpp/include/cudf/table/table_view.hpp

```diff
@@ -38,13 +38,14 @@ namespace detail {
  * @brief Base class for a table of `ColumnView`s
  *
  * This class should not be used directly. Instead:
- * -`table_view` should be used for a table of `column_view`s
- * -`mutable_table_view` should be used for a table of `mutable_column_view`s
+ * - `table_view` should be used for a table of `column_view` types
```

Comment:
I thought that `column_view` is just one "type", not multiple "types", isn't it? Similar for `mutable_column_view`.

Reply:
Sorry, I don't understand your question. 
The template variable `ColumnView` can be either `column_view` or `mutable_column_view`

Reply:
Oh I meant this:
```suggestion
 * - `table_view` should be used for a table of `column_view` type
 * - `mutable_table_view` should be used for a table of `mutable_column_view` type
```

Reply:
I think the David's intent with the plural was to reflect that the table contains multiple `column_view`s, so perhaps that's the better plural form?

Reply:
I just don't care for how that looks is all.

Reply:
I agree :slightly_smiling_face: Do whichever you like here, I was just trying to clear up any confusion. If we really want to be precise while still avoiding the ugliness, we could do something like "a table of `column_view` objects" instead.

Reply:
I updated the screen shot with my latest attempt.

Reply:
I like that version.